test/test_cuda.py

```diff
@@ -3432,6 +3432,34 @@ def test_cuda_device_memory_allocated(self):
         self.assertTrue(memory_allocated(0) > current_alloc[0])
         self.assertTrue(all(memory_allocated(torch.cuda.device(idx)) == current_alloc[idx] for idx in range(1, device_count)))
 
+    def test_matmul_memory_use(self):
+        def get_max_used():
+            torch.cuda.synchronize()
+            val = torch.cuda.max_memory_allocated()
+            torch.cuda.reset_peak_memory_stats()
+            return val
+
+        a = torch.rand(1, 32, 32, device="cuda")
+        b = torch.rand(24, 32, 1, device="cuda")
+
+        get_max_used()
```

Comment:
The return value of this call is ignored, are you using it to reset_peak_memory only?

Reply:
Yes.